### PR TITLE
chore(RelaseInfo): update to v5.17.0

### PR DESCRIPTION
**Pull request by marstamm**
image

### CODE REVIEW

File: client/src/plugins/version-info/ReleaseInfo.js [hunk 1]
```diff
@@ -44,20 +44,20 @@ export function ReleaseInfo(props) {
     <div className={ css.ReleaseInfo }>
       <ul className="dashed">
         <li>
-          <h4>Forms improvements</h4>
-          Added a group component and support for nested data (form keys).
+          <h4>Preview Append on Hover</h4>
+          Easily visualize your BPMN diagrams with our new feature. Hover over a context pad entry to see a preview of the created element.
```
Comment (christian-konrad) [inline client/src/plugins/version-info/ReleaseInfo.js:48]:
```suggestion
          Advance your BPMN diagrams with confidence using this new feature. Hover over a context pad entry to see a preview of the element to create.
```

**Review by christian-konrad — CHANGES_REQUESTED** [review]
Small wording change requested :)

File: client/src/plugins/version-info/ReleaseInfo.js (hunk 1 above)
Comment (marstamm) [reply client/src/plugins/version-info/ReleaseInfo.js]: adjusted with [9716e43](https://github.com/camunda/camunda-modeler/pull/3964/commits/9716e437c173cfab5277162bb7e9ce79eca5eb25)

File: client/src/plugins/version-info/ReleaseInfo.js [hunk 2]
```diff
@@ -48,7 +48,7 @@ export function ReleaseInfo(props) {
           Advance your BPMN diagrams with confidence using this new feature. Hover over a context pad entry to see a preview of the element to create.
         </li>
         <li>
-          <h4>Form Deployment to Camunda 8</h4>
+          <h4>Support for Upcoming Camunda 8 Features - Standalone Form Deployment and Linking</h4>
```
Comment (nikku) [inline client/src/plugins/version-info/ReleaseInfo.js:51]:
```suggestion
          <h4>Standalone Form Deployment and Linking</h4>
```

Comment (nikku) [inline client/src/plugins/version-info/ReleaseInfo.js:51]:
```suggestion
          <h4>Standalone Form Deployment and Linking</h4>
```

File: client/src/plugins/version-info/ReleaseInfo.js [hunk 3]
```diff
@@ -48,7 +48,7 @@ export function ReleaseInfo(props) {
...
           You can now deploy forms directly from the forms tab to Camunda 8. To use the deployed form, you can reference it in your BPMN diagram as a linked form.
```
Comment (nikku) [inline client/src/plugins/version-info/ReleaseInfo.js:52]:
```suggestion
You can now deploy forms directly from the forms tab. Reference it from a user task inside a BPMN diagram and Tasklist will pick it up for task completion.
```

Comment (nikku) [reply client/src/plugins/version-info/ReleaseInfo.js:52]: Or something like: _Supporting upcoming Camunda 8 features, ..._

Comment (marstamm) [reply client/src/plugins/version-info/ReleaseInfo.js]: I tried to implement the suggestions in [93017ba](https://github.com/camunda/camunda-modeler/pull/3964/commits/93017ba2343fa38871b46168ec346b81f2c281c8), let me know what you think

Comment (nikku) [reply client/src/plugins/version-info/ReleaseInfo.js]: Great :clap:

**Review by nikku — APPROVED** [review]

**Pull request merged**